internal/controllers/scheme.go

```diff
@@ -0,0 +1,17 @@
+package controllers
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
+	corev1alpha1 "github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+)
+
+func NewScheme() *runtime.Scheme {
```

Comment:
Ah, yep, it's on `ExtraConfig`. I'm hitting a funky issue though, not sure if we can share schemes between the actual API server and controller manager:
```
error initializing *filewatch.Controller controller: multiple group-version-kinds associated with type *v1alpha1.FileWatch, refusing to guess at one
```
Comes from https://github.com/kubernetes-sigs/controller-runtime/blob/e388e1e1f5a745166955442294bb8aede1f761e2/pkg/client/apiutil/apimachinery.go#L108-L113

The two versions are `__internal` and `v1alpha1`. The former comes from the API server's scheme registration:
https://github.com/tilt-dev/tilt-apiserver/blob/095950403c2cbc559e2f201da1c22e66b425e202/pkg/server/builder/resource/types.go#L119-L120

(Since it's kind of weird/indirect, when there are multi versions of an entity, the storage version is always going to get the `__internal` version in API server scheme.)

LMK if I'm missing something or doing something wrong - I am not familiar with the depths of the API server

Reply:
i don't know, let's just check this in and i can debug what's going on...